packages/devtools-launchpad/.babelrc

```diff
@@ -10,7 +10,7 @@
 
     ["module-resolver", {
       "alias": {
-        "react-dom": "react-dom/dist/react-dom",
+        "react-dom": "react-dom/umd/react-dom.development",
```

Comment:
Here was the warning we saw:
https://github.com/devtools-html/debugger.html/issues/1640

And the _fix_:
https://github.com/devtools-html/debugger.html/issues/1731

We've had several other issues creep in over time
https://github.com/devtools-html/devtools-core/issues/179
https://github.com/devtools-html/devtools-core/issues/23